src/index.html

```diff
@@ -7,8 +7,29 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo"><img src="images/logo.png" alt="LOGO"></a>
```

Comment:
```suggestion
      <a href="#" class="logo">
        <img src="images/logo.png" alt="LOGO">
      </a>
```
Новый тег - новая линия